lib/ur/usage_report.go

```diff
@@ -33,7 +34,12 @@ import (
 // Current version number of the usage report, for acceptance purposes. If
 // fields are added or changed this integer must be incremented so that users
 // are prompted for acceptance of the new report.
-const Version = 3
+const Version = 4
```

Comment:
I know it's easy to piggyback of this, but the "diff" thing allowing to diff between the Ur versions will show no diffs, causing utter confusion.

I guess it's sad to duplicate all of the code, but I feel we should give some sort of indication there, or have a separate prompt. This also could be enabled by default with some warning on startup?